src/screens/Followings/elements/TopicScreen/SuggestionTopic.js

```diff
@@ -27,7 +27,7 @@ const styles = StyleSheet.create({
   },
   moreText: {
     fontSize: 14,
-    color: colors.bondi_blue
+    color: COLORS.holyTosca
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, replacing hardcoded color values with references to color constants from a theme file is a good practice for maintainability.

```diff
- color: colors.bondi_blue
+ color: COLORS.holyTosca
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->